src/domains/project/dto/index.ts

```diff
@@ -172,6 +173,44 @@ export enum ProjectRole {
   DEVELOPER = 'Developer',
 }
 
+/**
+ * Represents the project name for the project filter.
+ */
+export class KeyPhraseParamDTO {
+  @IsString()
+  @IsNotEmpty()
+  projectName?: string;
```

Comment:
Consider trimming the input with a transform annotation or in the controller.

Reply:
done